[2022-09-04T16:52:50.029+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.get_datetime manual__2022-09-04T16:52:49.268244+00:00 [queued]>
[2022-09-04T16:52:50.039+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.get_datetime manual__2022-09-04T16:52:49.268244+00:00 [queued]>
[2022-09-04T16:52:50.040+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-04T16:52:50.041+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-09-04T16:52:50.042+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-04T16:52:50.051+0000] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): get_datetime> on 2022-09-04 16:52:49.268244+00:00
[2022-09-04T16:52:50.057+0000] {standard_task_runner.py:52} INFO - Started process 103 to run task
[2022-09-04T16:52:50.062+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_***_dag', 'get_datetime', 'manual__2022-09-04T16:52:49.268244+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpjb07y7_b', '--error-file', '/tmp/tmp_387st2w']
[2022-09-04T16:52:50.064+0000] {standard_task_runner.py:80} INFO - Job 41: Subtask get_datetime
[2022-09-04T16:52:50.126+0000] {task_command.py:371} INFO - Running <TaskInstance: first_airflow_dag.get_datetime manual__2022-09-04T16:52:49.268244+00:00 [running]> on host 16715446b510
[2022-09-04T16:52:50.177+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_***_dag
AIRFLOW_CTX_TASK_ID=get_datetime
AIRFLOW_CTX_EXECUTION_DATE=2022-09-04T16:52:49.268244+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-04T16:52:49.268244+00:00
[2022-09-04T16:52:50.178+0000] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-09-04T16:52:50.180+0000] {subprocess.py:74} INFO - Running command: ['/bin/bash', '-c', 'date']
[2022-09-04T16:52:50.191+0000] {subprocess.py:85} INFO - Output:
[2022-09-04T16:52:50.194+0000] {subprocess.py:92} INFO - Sun Sep  4 16:52:50 UTC 2022
[2022-09-04T16:52:50.195+0000] {subprocess.py:96} INFO - Command exited with return code 0
[2022-09-04T16:52:50.223+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=first_***_dag, task_id=get_datetime, execution_date=20220904T165249, start_date=20220904T165250, end_date=20220904T165250
[2022-09-04T16:52:50.274+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-04T16:52:50.311+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
